compiler/sc5-in.scp

```diff
@@ -274,4 +274,5 @@ static const char *warnmsg[] = {
 /*238*/  "'%s:' is an illegal cast; use view_as<%s>(expression)\n",
 /*239*/  "'%s' is an illegal tag; use %s as a type\n",
 /*240*/  "'%s:' is an old-style tag operation; use view_as<%s>(expression) instead\n",
+/*241*/  "user warning: %s\n",
```

Comment:
Can we use one of the unused ones?

Reply:
I don't see why not, is it generally preferred to use the lowest unused spot?

Reply:
Ya, preferred, but isn't super important.

Reply:
Patching holes is definitely a nice to have.

Reply:
Will do. I’ll probably commit in the coming days